pkg/crc/logging/logging.go

```diff
@@ -30,6 +30,30 @@ func CloseLogFile() {
 	logfile.Close()
 }
 
+func GetLatestContentOfLogFile(path string) ([]byte, error) {
```

Comment:
This is where we need to create a new interface/struct otherwise it will be impossible to test.
It will also help you in the "crc delete deletes also logs" PR I guess as you will be able to mock the log system.